packages/gnome-shell/src/extensions/global.d.ts

```diff
@@ -36,6 +36,70 @@ declare global {
     }
 }
 
+declare module '@girs/gobject-2.0/gobject-2.0' {
+    export namespace GObject {
+        interface Object {
+            /**
+             * Connect one or more signals, and associate the handlers
+             * with a tracked object.
+             *
+             * All handlers for a particular object can be disconnected
+             * by calling disconnectObject(). If object is a {Clutter.widget},
+             * this is done automatically when the widget is destroyed.
+             *
+             * @param {...any} args - a sequence of signal-name/handler pairs
```

Comment:
Please do not include type information in the doc comments in Typescript.  At best, it's redundant, at worst, wrong and confusing 🙂